src/expressions.md

```diff
@@ -276,7 +276,7 @@ They are never allowed before:
 
 [assign]:               expressions/operator-expr.md#assignment-expressions
 [borrow]:               expressions/operator-expr.md#borrow-operators
-[addr-of]:              expressions/operator-expr.md#raw-address-of-operators
+[raw-borrow]:           expressions/operator-expr.md#raw-borrow-operators
```

Comment:
If you would, just make the anchor "raw borrow" and correspondingly simplify the use of it above, and move this line to its alphabetized spot (even though it muddies up the diff).